narwhals/_arrow/series.py

```diff
@@ -340,6 +340,28 @@ def item(self: Self, index: int | None = None) -> Any:
             return self._native_series[0].as_py()
         return self._native_series[index].as_py()
 
+    def value_counts(self: Self, *, sort: bool = False, parallel: bool = False) -> Any:  # noqa: ARG002
+        """Parallel is unused, exists for compatibility"""
+        from narwhals._arrow.dataframe import ArrowDataFrame
+
+        pc = get_pyarrow_compute()
+        pa = get_pyarrow()
+
+        name_ = (
+            "index" if self._native_series._name is None else self._native_series._name
+        )
+
+        val_count = pc.value_counts(self._native_series)
+        val_count = pa.Table.from_struct_array(val_count).rename_columns([name_, "count"])
```

Comment:
looks like old versions of pyarrow don't have `from_struct_array`, seems it was only added in PyArrow 15.0.0 https://arrow.apache.org/release/15.0.0.html

maybe here we need to do a check on `self._backend_version` and raise `NotImplementedError` if `self._backend_version` is older than 15.0.0?

Reply:
@FBruzzesi  also suggested we could unpack the struct manually
```
value_counts = pc.value_counts(series)

pa.Table.from_arrays([
    value_counts.field("values"),
    value_counts.field("counts")
    ], names=["name_", "count])
```

WDYT?

Reply:
ooh that'd be better if possible, nice!

Reply:
okay, now I realised that I use ```
    if **"pandas_series_nullable"** in str(constructor) and sys.version_info < (
        3,
        9,
    ):  # fails for py3.8
        request.applymarker(pytest.mark.xfail)
```
while using a dataframe constructor, could this be the reason for failing 3.8v checks?
If that's the case should I change it to "pandas_nullable_constructor"?

Reply:
Yes! It is not marking anything to fail, thus pytest does not expect any test to be failing!

> If that's the case should I change it to "pandas_nullable_constructor"?

More yes!

Reply:
all green 🥦🥦🥦🥦🥦

Reply:
😆 best "all green" emoji ever